### PR TITLE
DOC-6180: Index GUI improvements

### PR DESCRIPTION
**Pull request by simon-dew**
The following documentation is ready for review:

* [Manage Indexes](https://simon-dew.github.io/docs-site/DOC-6180/server/6.5/manage/manage-indexes/manage-indexes.html)
* [Monitor Indexes](https://simon-dew.github.io/docs-site/DOC-6180/server/6.5/manage/monitor/monitoring-indexes.html)
* [Couchbase Web Console](https://simon-dew.github.io/docs-site/DOC-6180/server/6.5/manage/manage-ui/manage-ui.html) — removed some index info and added it to new pages above

Docs issue: [DOC-6180](https://issues.couchbase.com/browse/DOC-6180)

### CODE REVIEW

**Review by tonyjhillman — COMMENTED** [review]
Thanks, Simon. A couple of things I would mention. One is that throughout the Manage section, which contains UI examples on almost every page, I have not use the .btn graphic, whereby the name of the button is both emboldened and surrounded with square brackets and space - for example, [ Save ]. I have just emboldened the word and have referred to 'the Save button'. So this page, in using the square brackets, wouldn't be consistent with that. My reason for not using, I guess, has been twofold: one is I don't like the effect; it doesn't seem visually helpful, and doesn't look elegant. The other is that we have all kinds of other things to click on, like radio buttons, information symbols, links, tabs, and we don't do anything special in text for them. So for simplicity, I have not used this effect on references to buttons.

The other thing is that when possible (though not with absolute consistency), I have tried to put, on each page in Manage, a section for UI, CLI, and REST API. I'm not sure if some or all of the things you are doing with the UI could also be done with, say, the REST API: but if they could, would it be possible to add a section showing how to perform the equivalent tasks with REST? Perhaps it wouldn't work, but I thought it was worth asking.

File: modules/manage/pages/manage-indexes/manage-indexes.adoc [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+
+[[index-summary]]
+== Index Summary
+
+The index list displays the following information about each index:
+
+index name::
+The name of the index or replica.
++
+There may also be indicators after the index name, giving further information:
```
Comment (simon-dew) [inline modules/manage/pages/manage-indexes/manage-indexes.adoc]: insert "one or more"

File: modules/manage/pages/manage-indexes/manage-indexes.adoc [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+
+* `partitioned` indicates that the index is {index-partitioning}[partitioned].
+* `replica __n__` indicates that this is an {index-replication}[index replica], where `__n__` is the replica ID.
+* `stale` indicates that the node on which the index or partition is stored is not available.
+
+requests/sec::
+The number of requests per second.
+
+resident ratio::
+Percentage of the data held in memory.
```
Comment (simon-dew) [inline modules/manage/pages/manage-indexes/manage-indexes.adoc]: Insert "The"

File: modules/manage/pages/monitor/monitoring-indexes.adoc [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+
+Index Cache Miss Ratio::
+The percentage of accesses made to this index from disk, rather than from RAM.
+
+[[index-stats-interval]]
+=== Change the Statistics Interval
+
+To change the interval over which the statistics are displayed:
+
+. Open the drop-down list to the right of the *Index Stats* heading
```
Comment (simon-dew) [inline modules/manage/pages/monitor/monitoring-indexes.adoc]: Add full stop

**Comment by simon-dew** [comment]
- [x] Need link from [Monitor Statistics](https://docs.couchbase.com/server/6.0/manage/monitor/ui-monitoring-statistics.html) to the monitoring indexes page.
- [x] Also need to make clear that the individual index statistics are the same as the statistics shown in the **Index Stats** page.

**Comment by simon-dew** [comment]
> The other thing is that when possible (though not with absolute consistency), I have tried to put, on each page in Manage, a section for UI, CLI, and REST API

Thanks @tonyjhillman. There isn't always a straightforward equivalent in the CLI or the REST API for the functionality that's available through the Indexes screen, but where there is, I'll try to make it clearer.

**Comment by simon-dew** [comment]
> throughout the Manage section, which contains UI examples on almost every page, I have not use the .btn graphic

Fixed

**Comment by simon-dew** [comment]
> would it be possible to add a section showing how to perform the equivalent tasks with REST?

Done

**Review by tonyjhillman — APPROVED** [review]

File: modules/manage/pages/manage-indexes/manage-indexes.adoc [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+=== Show the Index Statistics
+
+To see statistics for the index, click the *Index Stats* heading in the expanded index row.
+
+The *Index Stats* heading is a toggle -- click the heading again to hide the statistics.
+
+For details of the index statistics, refer to {index-stats}[Index Statistics].
+
+[[edit-index]]
+=== Edit the Index
```
Comment (deepkaran) [inline modules/manage/pages/manage-indexes/manage-indexes.adoc]: "Edit the index" can be a little confusing. "Open in workbench" shows the index definition in the query workbench. A user can modify the definition and create a new index using the changed definition but cannot "edit" the existing index. The only way to "edit" an index is using Alter Index command (placement and replica count).

**Pull request merged**